src/test/ui/suggestions/multibyte-escapes.rs

```diff
@@ -0,0 +1,18 @@
+// Regression test for #87397.
```

Comment:
Could add `// run-rustfix` to this test.

Reply:
Unfortunately not, because the second error is not automatically fixable.